templates/projects/list.tmpl

```diff
@@ -38,30 +38,41 @@
 		<div class="milestone list">
 			{{range .Projects}}
 				<li class="item">
-					{{svg .IconName}} <a href="{{.Link}}">{{.Title}}</a>
-					<div class="meta">
-						{{$closedDate:= TimeSinceUnix .ClosedDateUnix $.locale}}
-						{{if .IsClosed}}
-							{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.closed" $closedDate | Safe}}
-						{{end}}
-						<span class="issue-stats">
-							{{svg "octicon-issue-opened" 16 "gt-mr-3"}}
-							{{$.locale.PrettyNumber .NumOpenIssues}}&nbsp;{{$.locale.Tr "repo.issues.open_title"}}
-							{{svg "octicon-check" 16 "gt-mr-3"}}
-							{{$.locale.PrettyNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
-						</span>
+					<div class="header">
+						<h3 class="gt-df gt-ac gt-m-0 gt-fw">
+							{{svg .IconName 16 "gt-mr-3"}}
+							<a class="muted" href="{{.Link}}">{{.Title}}</a>
+						</h3>
 					</div>
-					{{if and $.CanWriteProjects (not $.Repository.IsArchived)}}
-					<div class="ui right operate">
-						<a href="{{$.Link}}/{{.ID}}/edit" data-id={{.ID}} data-title={{.Title}}>{{svg "octicon-pencil"}} {{$.locale.Tr "repo.issues.label_edit"}}</a>
-						{{if .IsClosed}}
-							<a class="link-action" href data-url="{{$.Link}}/{{.ID}}/open">{{svg "octicon-check"}} {{$.locale.Tr "repo.projects.open"}}</a>
-						{{else}}
-							<a class="link-action" href data-url="{{$.Link}}/{{.ID}}/close">{{svg "octicon-skip"}} {{$.locale.Tr "repo.projects.close"}}</a>
+					<div class="body">
```

Comment:
What is "body"? Is it defined by Fomantic UI? If not, it could have at least two words, for maintainability.

https://docs.gitea.com/contributing/guidelines-frontend